### DISCUSSION TITLE
QRCoder Implementation

### DISCUSSION THREAD

**Discussion by MdAli-dev** [Shane32/QRCoder#305] [Ideas]
I have used QRCoder @ my new QR Code generator website [www.qr-code-wizard.com](https://www.qr-code-wizard.com/). Thank you it's really wonderful.

**Comment by SalesGarden** [comment]
Hello - a quick question, if you please.  We are going to implement a QR Code Generator into our project.  I saw the nice job you did with the QR Code Wizard.  How long did/should it take to develop from the GitHub code?  Thanks!

**Reply by GirishSolanki** [reply]
It will take one day to integrate.

**Reply by MdAli-dev** [reply]
Hi,

It depends on UI. it took 2 days to complete "Create QR" section including UI. 

Thanks

**Comment by SalesGarden** [comment]
Thanks for the information.  Much appreciated!

**Comment by SalesGarden** [comment]
Thanks for the information.  Much appreciated!
